recipes/vtku3dexporter/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: vtku3dexporter
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ClinicalGraphics/u3d.git
+  git_tag: {{ version }}
+
+build:
+  features:
+  - vc9  # [win and py27]
+  - vc10  # [win and py34]
+  - vc14  # [win and py35]
+  number: 0
+  skip: true  # [osx or win32 or linux32]
```

Comment:
Because we don't have anyone who uses OSX, but if someone knows how the build.sh would have to be patched, it could be re-enabled. It might work for win32, but do people actually still need that?